src/composables/usePool.ts

```diff
@@ -357,6 +357,30 @@ export function fiatValueOf(pool: Pool, shares: string): string {
   return bnum(shares).times(bptPriceFor(pool)).toString();
 }
 
+export function findTokenByAddress(pool: Pool, address: string) {
+  return pool.tokens.find(token => isSameAddress(token.address, address));
+}
```

Comment:
This doesn't really work. It only finds tokens in layer 1 of the token tree. You need to search through all nested layers. I added a function for this, in this PR: https://github.com/balancer-labs/frontend-v2/pull/2404/files#diff-c7a3eec563f69ad2832ed59c9eda57a3a98acab763483eaa6ef08e401bd8f605R356-R370

It will be merged to the feature base soon so you should be able to use it.